.eslintrc.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
         'airbnb-typescript/base',
     ],
     rules: {
-        "@typescript-eslint/indent": ["error", 4]
+        '@typescript-eslint/indent': ['error', 4],
+        'arrow-parens': ['error', 'as-needed']
```

Comment:
why has this been changed?

Reply:
@lukasz-kowal introduced indent rule.

I introduced arrow-parens because linter was complaining about functions with single parameter not being wrapped in parentheses.